src/common/commonutils/FileUtils.c

```diff
@@ -1199,6 +1198,16 @@ int CheckTextIsNotFoundInFile(const char* fileName, const char* text, char** rea
     return result;
 }
 
+static bool IsValidGrepArgument(const char* text)
+{
+    return IsValidDaemonName(text);
+}
+
+static bool IsValidCommentCharacter(char c)
+{
+    return ((0 == isalnum(c)) && ('#' != c)) ? false : true;

```

Comment:
Could we do sth like this maybe?
`return isalnum(c) || (c == '#');`
This looks more clear to me.

Reply:
Yes!

Reply:
// Some of the common comment characters that can be encountered, add more if necessary
static bool IsValidCommentCharacter(char c)
{
    return (('#' == c) || ('/' == c) || ('*' == c) || (';' == c) || ('!' == c)) ? true : false;
}